cmd/admin-handlers-users_test.go

```diff
@@ -896,13 +897,13 @@ func (s *TestSuiteIAM) TestServiceAccountOpsByUser(c *check) {
 	// 3. Check S3 access
 	c.assertSvcAccS3Access(ctx, s, cr, bucket)
 
-	// 4. Check that svc account can restrict the policy, and that the
-	// session policy can be updated.
-	c.assertSvcAccSessionPolicyUpdate(ctx, s, userAdmClient, accessKey, bucket)
+	// // 4. Check that svc account can restrict the policy, and that the
```

Comment:
Just delete all these. We have git history if we need to find them again.

Reply:
Yep.